evennia/server/server.py

```diff
@@ -492,6 +500,7 @@ def at_server_start(self):
         This is called every time the server starts up, regardless of
         how it was shut down.
         """
+        evennia.PLUGINS.at_server_start()
```

Comment:
Should this really run _before_ the server-level `at_server_start`? I'd imagine you'd want the plugin to modify/operate on/extend what the server itself does?